crates/bevy_pbr/src/render/pbr_prepass.wgsl

```diff
@@ -90,18 +108,25 @@ fn fragment(in: FragmentInput) -> @location(0) vec4<f32> {
 #endif // VERTEX_UVS
         );
 
-        return vec4(normal * 0.5 + vec3(0.5), 1.0);
+        out.normal = vec4(normal * 0.5 + vec3(0.5), 1.0);
     } else {
-        return vec4(in.world_normal * 0.5 + vec3(0.5), 1.0);
+        out.normal = vec4(in.world_normal * 0.5 + vec3(0.5), 1.0);
     }
-}
+#endif // NORMAL_PREPASS
 
-#else // NORMAL_PREPASS
+#ifdef MOTION_VECTOR_PREPASS
+    let clip_position_t = view.unjittered_view_proj * in.world_position;
+    let clip_position = clip_position_t.xy / clip_position_t.w;
+    let previous_clip_position_t = previous_view_proj * in.previous_world_position;
+    let previous_clip_position = previous_clip_position_t.xy / previous_clip_position_t.w;
+    out.motion_vector = (clip_position - previous_clip_position) * vec2(0.5, -0.5);
```

Comment:
I see this code is duplicated from prepass.wgsl. I wonder if it's worth putting it into a utility function to keep them in sync?

Reply:
All of prepass/pbr_prepass is duplicated, basically. I'm not sure a function would help, it's just something to be careful about.